Resfeber/View Controllers/SideMenuController.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  SideMenuController.swift
+//  Resfeber
+//
+//  Created by David Wright on 11/17/20.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import UIKit
+
+private let reuseIdentifer = "MenuOptionCell"
+
+enum MenuOption: Int, CustomStringConvertible, CaseIterable {
+    
+    case EditProfile
+    case LogOut
+    
+    var description: String {
+        switch self {
+        case .EditProfile: return "Edit Profile"
+        case .LogOut: return "Log Out"
+        }
+    }
+    
+    var image: UIImage? {
+        switch self {
+        case .EditProfile: return UIImage(systemName: "person")?.withTintColor(UIColor.Resfeber.light, renderingMode: .alwaysOriginal)
+        case .LogOut: return UIImage(systemName: "escape")?.withTintColor(UIColor.Resfeber.light, renderingMode: .alwaysOriginal)
+        }
+    }
+}
+
+protocol SideMenuDelegate: class {
+    func toggleSideMenu(withMenuOption menuOption: MenuOption?)
+}
+
+class SideMenuController: UIViewController {
+    
+    // MARK: - Properties
+    
+    var profile: Profile = ProfileController.shared.profile
+    var tableView: UITableView!
+    weak var delegate: SideMenuDelegate?
+    
+    private lazy var profileMenuHeader: SideMenuHeader = {
+        let frame = CGRect(x: 0, y: 0, width: self.view.frame.width - 80, height: 140)
+        let view = SideMenuHeader(profile: profile, frame: frame)
+        return view
+    }()
+    
+    // MARK: - Lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = UIColor.Resfeber.red
```

Comment:
So I mentioned this in another PR, but maybe the feedback didn't get implemented. What I would do instead of `UIColor.Resfeber.red` is I would create a completely new class called `ResfeberColor` or `RFColor` and make it a subclass of `UIColor` (ie: `RFColor: UIColor`), and that way whenever you want to use one of your custom colors, you simply call `RFColor.red`